lib/actions/config.js

```diff
@@ -152,24 +151,22 @@ Examples:
 	],
 	permission: 'user',
 	root: true,
-	action(params, options) {
-		const Bluebird = require('bluebird');
-		const config = require('balena-config-json');
-		const umountAsync = Bluebird.promisify(require('umount').umount);
+	async action(params, options) {
+		const umountAsync = await getUmountAsync();
 
-		return Bluebird.try(
-			() => options.drive || getVisuals().drive('Select the device drive'),
-		)
-			.tap(umountAsync)
-			.then((drive) =>
-				require('fs')
-					.promises.readFile(params.file, 'utf8')
-					.then(JSON.parse)
-					.then((configJSON) => config.write(drive, options.type, configJSON)),
-			)
-			.tap(() => {
-				console.info('Done');
-			});
+		const drive =
+			options.drive || getVisuals().drive('Select the device drive');
```

Comment:
Is it missing `await` here, before `getVisuals().drive(...)`? I image it would return a promise...